pkg/webhook/resources/setting/validator.go

```diff
@@ -536,7 +536,7 @@ func validateNTPServers(setting *v1beta1.Setting) error {
 		logrus.Errorf(errMsg)
 		return fmt.Errorf(errMsg)
 	}
-	startWithHTTP, err := regexp.Compile("^http?://.*")
+	startWithHTTP, err := regexp.Compile("^https?://.*")
```

Comment:
I'm just curious about that. Does the NTP server need `https`?

If we use the local NTP server, could it pass?

Reply:
> I'm just curious about that. Does the NTP server need https?

Users should use the FQDN format for NTP server settings. The validator will throw an error if the server address starts with http or https.

> If we use the local NTP server, could it pass?

Did you mean setting up a local NTP server, passing its address to the settings, and then checking timedatectl timesync-status to ensure it works?

Reply:
Ah..., yes, I misunderstood something here.
There is no issue.

Thanks for it.